OpenEdXMobile/src/main/java/org/edx/mobile/view/CourseDashboardFragment.java

```diff
@@ -219,6 +216,46 @@ public void onMenuItemClick(@NonNull ComponentName componentName, @NonNull Share
                         intent.setComponent(componentName);
                         startActivity(intent);
                     }
+
+                    @NonNull
+                    private String getFacebookSharingText() {
+                        String shareText;
+                        final String utmParams = courseData.getCourse().getCourseSharingUtmParams("facebook");
```

Comment:
The utm param keys should be standardised in an already defined enum i.e. `org.edx.mobile.util.images.ShareUtils.ShareType` to avoid typos.